examples/stable/test/property/canister_methods/on_low_wasm_memory/dfx.json

```diff
@@ -0,0 +1,13 @@
+{
+    "canisters": {
+        "canister": {
+            "type": "azle",
+            "main": "src/index.ts",
+            "candid_gen": "automatic",
```

Comment:
Why is `"candid_gen": "automatic"` here? I believe this should be removed from all of the stable property tests if it is present. That doesn't even do anything in that location anymore, as it would have to be in the `"custom"` property.

Reply:
Done